conda_forge_tick/utils.py

```diff
@@ -761,6 +761,14 @@ def populate_feedstock_attributes(
                 ),
             )
 
+            # sometimes the requirements come out to None and this ruins the
+            # aggregated meta_yaml
+            if "requirements" in varient_yamls[-1]:
+                for section in ["build", "host", "run"]:
+                    if section in varient_yamls[-1]["requirements"]:
```

Comment:
We make sure to set a section only if it is actually in the recipe. Adding a section when it is not there might confuse migrators trying to move CB2 recipes to CB3.